extensions/guacamole-auth-radius/src/main/java/org/apache/guacamole/auth/radius/RadiusConnectionService.java

```diff
@@ -68,6 +70,27 @@
     private ConfigurationService confService;
 
 
+    /**
+     * 
```

Comment:
Empty comment.

Reply:
Not seeing this in the current set of commits...were you looking at an earlier commit??